package.json

```diff
@@ -37,13 +37,13 @@
     "@native-html/table-plugin": "2.1.5",
     "@react-native-async-storage/async-storage": "~1.17.3",
     "@react-native-community/datetimepicker": "6.1.2",
-    "@react-native-community/masked-view": "0.1.10",
     "@react-native-community/netinfo": "8.2.0",
+    "@react-native-masked-view/masked-view": "0.2.6",
     "@react-navigation/bottom-tabs": "5.11.15",
     "@react-navigation/drawer": "5.12.9",
     "@react-navigation/native": "5.9.8",
     "@react-navigation/stack": "5.14.9",
-    "@sentry/react-native": "^3.2.10",
+    "@sentry/react-native": "^3.1.1",
     "@viro-community/react-viro": "^2.23.0",
```

Comment:
What is the reason for the version downgrade?

Reply:
this in yarn.lock

<img width="902" alt="Bildschirmfoto 2022-07-22 um 11 57 12" src="https://user-images.githubusercontent.com/1942953/180415254-1e8543a7-e09d-4468-af15-9efb989f078d.png">

it is versioned more accordingly now

Reply:
Thank you. 👍